analyzers/tests/SonarAnalyzer.UnitTest/Rules/MemberInitializedToDefaultTest.cs

```diff
@@ -25,21 +25,26 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class MemberInitializedToDefaultTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<MemberInitializedToDefault>();
+
         [TestMethod]
         public void MemberInitializedToDefault() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\MemberInitializedToDefault.cs", new MemberInitializedToDefault());
+            builder.AddPaths(@"MemberInitializedToDefault.cs").Verify();
 
 #if NET
+
         [TestMethod]
         public void MemberInitializedToDefault_CSharp9() =>
-            OldVerifier.VerifyAnalyzerFromCSharp9Library(@"TestCases\MemberInitializedToDefault.CSharp9.cs", new MemberInitializedToDefault());
+            builder.AddPaths(@"MemberInitializedToDefault.CSharp9.cs").WithOptions(ParseOptionsHelper.FromCSharp9).Verify();
+
 #endif
 
         [TestMethod]
         public void MemberInitializedToDefault_CodeFix() =>
-            OldVerifier.VerifyCodeFix<MemberInitializedToDefaultCodeFix>(
-                @"TestCases\MemberInitializedToDefault.cs",
-                @"TestCases\MemberInitializedToDefault.Fixed.cs",
-                new MemberInitializedToDefault());
+            builder
```

Comment:
In some cases we let the first invocation be on the first line, and at some places we  put all invocations to separate lines, from our coding style description it is not super clear to me what should be the way to go?

Reply:
Indeed. For me, the code is easier to read when all method invocations are aligned. We could discuss and make this explicit in our guidelines.